Makefile

```diff
@@ -16,17 +12,19 @@ MANDIR=$(PREFIX)/man/man8
 MKDIR=mkdir
 
 PROG=	iocage
+SCRIPTS=iocage
+SCRIPTSDIR=${PREFIX}/BINDIR
 MAN=	$(PROG).8
 
+${PROG}:
+	@echo Nothing needs to be done for iocage.
+
 install:
 	$(MKDIR) -p $(BINDIR)
 	$(MKDIR) -p $(FILESDIR)
-	$(INSTALL) -m $(BINMODE) $(PROG) $(BINDIR)/
-	$(INSTALL) lib/* $(FILESDIR)/
-	$(INSTALL) rc.d/* $(RCDIR)/
-	rm -f $(MAN).gz
-	gzip -k $(MAN)
-	$(INSTALL) $(MAN).gz $(MANDIR)/
-	rm -f $(MAN).gz
+	$(INSTALL) -c -m $(BINMODE) ${.OBJDIR}/$(PROG) $(BINDIR)/
+	$(INSTALL) -c ${.OBJDIR}/lib/* $(FILESDIR)/
+	$(INSTALL) -c ${.OBJDIR}/rc.d/* $(RCDIR)/
+	$(INSTALL) -c $(MAN).gz $(MANDIR)/
```

Comment:
Yep!  That's already done by bsd.prog.mk (which includes bsd.man.mk), so that bit was redundant: